app/lib/common/utils/utils.dart

```diff
@@ -266,12 +269,43 @@ List<String> asDartStringList(FfiListFfiString data) {
   return data.toList().map((e) => e.toDartString()).toList();
 }
 
+// check if string contains html tags
+bool isHTML(String str) {
+  final RegExp htmlRegExp =
+      RegExp('<[^>]*>', multiLine: true, caseSensitive: false);
+  return htmlRegExp.hasMatch(str);
+}
```

Comment:
This is generally not a good idea ... rather than trying to guess, the actual part that sent the stuff must know. So instead, that part should be specific.

```suggestion
```

Reply:
We have two specific methods that deal with pin content part .i.e. ```contentMarkdown()``` and ```contentHTML()``` which we use in ```updateBuilder``` for editing ```pinDraft``` for creating. And on our UI end, the singular controller that saves user description needs to know whether it's string that contains html or markdown. 
This boolean helps determining that scenario. I am open of having suggestions how to determine that and then sending either ```contentMarkdown()``` or ```contentHTML()```. Thoughts?